plugins/catalog/src/alpha/entityContents.tsx

```diff
@@ -37,7 +37,7 @@ export const OverviewEntityContent = createEntityContentExtension({
   },
   loader: async ({ inputs }) =>
     import('./EntityOverviewPage').then(m => (
-      <m.EntityOverviewPage cards={inputs.cards} />
+      <m.EntityOverviewPage cards={inputs.cards.map(c => c.output)} />
```

Comment:
yeah so there's a minor convenience cost here, this is probably gonna be somewhat common to write - then again, how often is the input type _exactly_ what goes into a prop? Maybe people already had to map `c => c.blah` and now instead do `c => c.output.blah`